MonkeyBot/Common/MonkeyHelpers.cs

```diff
@@ -19,8 +19,8 @@ public static async Task WriteTextAsync(string filePath, string text, bool appen
         {
             if (!File.Exists(filePath))
             {
-                string strippedPath = Path.GetDirectoryName(filePath);
-                if (!strippedPath.IsEmpty() && !Directory.Exists(strippedPath))
+                string? strippedPath = Path.GetDirectoryName(filePath);
+                if (strippedPath != null && !strippedPath.IsEmpty() && !Directory.Exists(strippedPath))
```

Comment:
IsEmpty() is a shorthand for string.IsNullOrEmpty(), I wonder if there's a way to make the analyzer pick that up?

Reply:
It might be fixed by changing the extension to extend a nullable string `IsEmpty(this string? s)`. I didn't look into it too deep, so I didn't catch that it also checked for null. I'm a strong proponent of verbose naming, so I just assumed it only checked for emptyness and not null. :-)

Reply:
Agreed on the naming, I was being inconsistent there :) Will change it later